fixtures/bookStoreFixture.ts

```diff
@@ -0,0 +1,5 @@
+import { BookStorePage } from '../POM/BookStorePage';
+import { Page } from '@playwright/test';
+
+
+export const bookStoreFixture = (page: Page) => new BookStorePage(page);
```

Comment:
this is not a fixture

https://playwright.dev/docs/test-fixtures#creating-a-fixture